app/frontend/common/components/footer/footer.less

```diff
@@ -0,0 +1,36 @@
+@import "../../colors";
+
+.footer {
+  background: @color-primaryDark;
+  border-top: solid 1px @color-accent;
+
+  ul {
+    margin: 0;
+    padding: 0;
+    li {
+      border-bottom: solid 0.1px @color-grayLight;
+    }
+    a {
+      display: block;
+      padding: 10px;
+
+      color: @color-white;
+    }
+  }
+
+  &-listItem-header {
+    font-size: .9rem;
+    font-weight: 400;
+    padding-bottom: .2rem;
+  }
+
+  &-listItem-text {
+    font-size: .6rem;
+    color: @color-grayLighter;
+  }
+}
+
+.arrow-icon {
+  width: 15px;
```

Comment:
Let's also resize the actual image to be at most 30px since we're optimizing for data usage.